json_merger/dict_merger.py

```diff
@@ -76,10 +76,11 @@ class SkipListsMerger(object):
     """3-way Merger that ignores list fields."""
 
     def __init__(self, root, head, update, default_op,
-                 data_lists=None):
+                 data_lists=None, custom_ops={}):
```

Comment:
Also would be nice to add the missing docstring to explain the structure of `custom_ops`.